pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -482,12 +482,12 @@ public CompletableFuture<Boolean> allowNamespaceOperationAsync(NamespaceName nam
                             case GET_TOPIC:
                             case GET_TOPICS:
                             case GET_BUNDLE:
+                            case CREATE_TOPIC:
                                 return allowConsumeOrProduceOpsAsync(namespaceName, role, authData);
```

Comment:
seems we're only checking permission for `consume` or `produce` action here, I'm wondering what's the benefit of this pr while we don't have a action for `create_topic` itself

Reply:
I see AuthorizationProviderBiscuit is check for namespace operator in `allowNamespaceOperationAsync`, but I don't see how moving up the `case CREATE_TOPIC:` in PulsarAuthorizationProvider is able solving the issue you described?
A client that subscribe to a topic that does not exist will still able to create the topic cause it has consume permission?

Reply:
It is not solving the issue. It is required to make the test pass to solve the issue. I could create a create_topic AuthAction, but it won't be backward compatible. WDYT?